### PR TITLE
expose public pulling image repository

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://github.com/openshift/origin-web-console/issues/1958

cc @spadgett

### CODE REVIEW

**Comment by spadgett** [comment]
@juanvallejo Wondering if we can't use the registry here:

https://github.com/cockpit-project/registry-image-widgets/blob/ad85aceeceaaa6c4adac8a273182a85e3518ddce/views/image-pull.html#L7-L8

@petervo PTAL

**Comment by petervo** [comment]
We can, but it should use an extra if so we can fall back to the setting if that object attribute is not available.

**Comment by spadgett** [comment]
@mfojtik FYI

File: views/imagestream-body.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
     <dd ng-if-end><tt>{{imagestream.spec.dockerImageRepository}}</tt></dd>
     <dt>Pulling repository</dt>
     <dd><tt>{{imagestream.status.dockerImageRepository}}</tt></dd>
+    <dt ng-if-start="imagestream.status.publicDockerImageRepository">Public pulling repo</dt>
```
Comment (thomasmckay) [inline views/imagestream-body.html]: Missing _translate_.
Can you link docs for what this attribute is to confirm nomenclature?
Thanks!

Comment (juanvallejo) [reply views/imagestream-body.html]: Not sure if we have public-facing docs yet, but here are godocs from the PR that introduced these fields: https://github.com/openshift/origin/pull/15853/files#diff-3788994f990386b63387ece5c3b5dc33 and https://github.com/openshift/origin/pull/15853/files#diff-1a35f433d29ab01f47e5563ac4933758R136

**Comment by juanvallejo** [comment]
@spadgett @petervo thanks updated `imagestream-push.html` and `image-pull.html` to make use of `publicDockerImageRepository` when field is available

**Review by petervo — APPROVED** [review]

**Review by spadgett — APPROVED** [review]

**Pull request merged**